go-controller/pkg/util/ovs.go

```diff
@@ -328,6 +328,44 @@ func GetOVSOfPort(args ...string) (string, string, error) {
 	return stdout, stderr, err
 }
 
+func GetDatapathType(bridge string) (string, error) {
+	br_type, err := GetOvsEntry("bridge", "br-int", "datapath_type", "")
+	if err != nil {
+		return "", err
+	}
+	return br_type, nil
+}
+
+// GetOvsEntry queries the OVS-DB using ovs-vsctl and returns
+// the requested entries.
+func GetOvsEntry(table, record, column, key string) (string, error) {
```

Comment:
no need to export this function since no one is using this outside of this package. I would just rename it to `getOvsEntry`

Reply:
ack